mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogStreamHeader.cs

```diff
@@ -41,6 +43,10 @@ internal LogStreamHeader (LogReader reader)
 
 			Version = new Version (reader.ReadByte (), reader.ReadByte ());
 			FormatVersion = reader.ReadByte ();
+
+			if (FormatVersion < MinimumMLPDSupportedVersion || FormatVersion > MaximumMLPDSupportedVersion)
+				throw new LogException ($"Unsupported MLPD version {FormatVersion}. Should be >= {MinimumMLPDSupportedVersion} and <= {MaximumMLPDSupportedVersion}");
```

Comment:
Missing period at the end of the message.

Reply:
done